benchmarks/transport_montecarlo_main_loop.py

```diff
@@ -21,7 +21,8 @@ def time_montecarlo_main_loop(self):
             self.transport_state.opacity_state,
             self.montecarlo_configuration,
             self.transport_state.radfield_mc_estimators,
-            self.transport_state.spectrum_frequency_grid.value,
+            self.spectrum_frequency_grid.value,
```

Comment:
These changes are likely to conflict with #2722

Reply:
I should leave it, and then when Ashish's PR is merged, I will rebase my PR with that.

Reply:
Even better, will make the changes that Ashish made to this file.